src/preact/base-element.js

```diff
@@ -121,9 +135,24 @@ export class PreactBaseElement extends AMP.BaseElement {
     this.context_.renderable = true;
     this.context_.playable = true;
     this.scheduleRender_();
+    if (this.observer_) {
+      this.observer_.observe(
+        this.element,
+        MUTATION_OBSERVER_OPTIONS.passthroughNonEmpty
```

Comment:
Then, this will be: `PASSTHROUGH_NON_EMPTY_MUTATION_INIT`

Reply:
Updated!